tf2onnx/rewriter/conv2d_with_add_rewriter.py

```diff
@@ -13,31 +13,39 @@
 # pylint: disable=missing-docstring
 
 def rewrite_biasadd_with_conv2d(g, ops):
-    pattern = \
+    pattern1 = \
         OpTypePattern('BiasAdd', name='biasadd', inputs=[
             OpTypePattern('Conv2D|Conv2DBackpropInput', name='conv', inputs=['*', '*']), '*'])
-    matcher = GraphMatcher(pattern)
-    match_results = list(matcher.match_ops(ops))
-    for match in match_results:
-        biasadd = match.get_op('biasadd')
-        conv = match.get_op('conv')
-
-        #backup the conv and biasadd values
-        conv_type = conv.type
-        conv_input = conv.input
-        conv_attr = conv.attr
-        dtype = g.get_dtype(conv.output[0])
-        shape = g.get_shape(conv.output[0])
-        conv_name = biasadd.name
-        conv_output = biasadd.output
-        conv_inputs = [conv_input[0], conv_input[1], biasadd.input[1]]
-
-        if len(g.find_output_consumers(conv.output[0])) > 1:
-            continue
-        # Remove the Conv and BiasAdd node
-        g.remove_node(conv.name)
-        g.remove_node(biasadd.name)
-
-        g.make_node(conv_type, conv_inputs, attr=conv_attr, name=conv_name, outputs=conv_output,
-                    shapes=[shape], dtypes=[dtype], skip_conversion=False)
+    pattern2 = \
```

Comment:
Hi @TomWildenhain-Microsoft , Could you help me to review this rewrite? I have tested the repro code from that issue, which can run successfully as expection. 
And do you have any idea about adding the test in somewhere about the `keras Conv2DTranspose` ?